.drone.yml

```diff
@@ -0,0 +1,35 @@
+---
+kind: pipeline
+type: docker
+name: default
```

Comment:
might want to rename to build so you can add more pipelines for tools/proxy etc.

Reply:
renamed to `main`